Resources/Prototypes/Roles/Jobs/Civilian/assistant.yml

```
@@ -11,6 +11,7 @@
   - Male: фрилансер
   - Male: экипаж вне службы
   - Male: турист
+  - Male: психотерапевт
```

```suggestion
  - Male: психолог
```
Тут должен быть психолог.